components/EntriesList.js

```diff
@@ -1,9 +1,6 @@
-import "./EntriesList.css";
-import { FavoriteButton } from "../FavoriteButton/FavoriteButton";
-
 export function EntriesList({ date, title, comment }) {
   return (
-    <div className="entries__section-entry">
+    <section className="entries__section-entry">
       <time className="entry-date">{date}</time>
```

Comment:
I guess we should use styled components for styling. Thats what the coaches said. Do you plan to implement it later?

Reply:
I thought at first I would need it. But as you can see in the code I did the styling with styled components and forgot to delete these classNames ;-)